Components/CorrectArrays/index.js

```diff
@@ -0,0 +1,16 @@
+export default function CorrectArrays(arrayofobjects) {
+  const [keyitems, ...csvArray] = arrayofobjects;
+  const csvArray2 = csvArray.filter(
+    (row, index, arr) => index !== arr.length - 1
+  );
```

Comment:
why are you doing this?

Reply:
The array of objects returned by convertCSVToArray has one row  too many at the end (a copy of the last row), so we remove it. We have now added comments in the CorrectArrays.